src/createComposable.ts

```diff
@@ -254,11 +253,6 @@ function createWatcher(
 
 function createPathGetter(ctx: any, path: string) {
   const segments = path.split('.')
-  return () => {
-    let cur = ctx
-    for (let i = 0; i < segments.length && cur; i++) {
-      cur = cur[segments[i]]
-    }
-    return cur
-  }
+  return () =>
+    ctx ? segments.reduce((cur, item) => (cur = cur[item]), ctx) : ctx
```

Comment:
KI'd prefer to keep the for loop. it's more expressive in general faster than reduce.

Reply:
of course